app/Routes.js

```diff
@@ -33,6 +33,7 @@ import AddressbookImpl from './utils/AddressbookImpl';
 import Sidebar from './components/Sidebar';
 import Transactions from './components/Transactions';
 import CompanionAppListener from './companion';
+import PastelID from './components/PastelID';
```

Comment:
I think we should change this to be Pastel_ID instead because it looks too similar to Pasteld

Reply:
Addressed according to conversation in Slack.